tests/views/test_survey.py

```diff
@@ -9,26 +9,27 @@
 from config import TestingConfig
 from response_operations_ui import app
 from response_operations_ui.controllers.survey_controllers import get_survey_short_name_by_id
+from response_operations_ui.views.surveys import _sort_collection_exercise
 
 url_get_survey_list = f'{app.config["BACKSTAGE_API_URL"]}/v1/survey/surveys'
 url_get_legal_basis_list = f'{app.config["SURVEY_URL"]}/legal-bases'
 url_create_survey = f'{app.config["SURVEY_URL"]}/surveys'
 
-with open('tests/test_data/survey/survey_list.json') as json_data:
-    survey_list = json.load(json_data)
-with open('tests/test_data/survey/legal_basis_list.json') as json_data:
-    legal_basis_list = json.load(json_data)
+with open('tests/test_data/survey/survey_list.json') as f:
```

Comment:
These should be as fp as @JamesGardiner mentioned in this PR: https://github.com/ONSdigital/ras-frontstage/pull/328#discussion-diff-187295508R27